taskcluster/ci/beetmover/kind.yml

```diff
@@ -17,14 +17,20 @@ kind-dependencies:
     - mac-notarization
     - repackage-signing
 
+task-defaults:
+    run-on-tasks-for: [action]
+    worker-type: beetmover
+    requires-level: 3
```

Comment:
Does this key mean, we can't test on level 1 against the staging repository? Might be nice to allow level 1, but automatically point at the staging deb repository in that case.

That said, consider this follow-up fodder, because this is how things already work with Android and I don't want to complicate this PR.